frontend/src/app/main/ui/viewer/inspect/exports.cljs

```diff
@@ -62,8 +64,14 @@
                                (cond-> share-id (assoc :share-id share-id)))
                   exports  (mapv #(merge % defaults) @exports)]
               (if (= 1 (count exports))
-                (st/emit! (de/request-simple-export {:export (first exports)}))
-                (st/emit! (de/request-multiple-export {:exports exports :filename filename}))))))
+                (st/emit!
+                 (de/request-simple-export {:export (first exports)})
+                 (ptk/event
+                  ::ev/event {::ev/name "export" :exports (str exports)}))
+                (st/emit!
+                 (de/request-multiple-export {:exports exports})
+                 (ptk/event
+                  ::ev/event {::ev/name "export" :exports (str exports)}))))))
```

Comment:
I think we should track the origin of the event, and use a more specific event name

Reply:
`export` is very generic and can be confused with exporting .zip or .penpot

Reply:
fixed